### ISSUE TITLE
cannot find file \Revive\Revive\ReviveInjector_x64.exe

### ISSUE THREAD

**Issue by glenbecker** [TheGreatCabbage/condor2-revive-helper#1]
Running the desktop shortcut, I see a momentary loading wheel (less than a second) and then nothing.
Normal injection works fine. I've confirmed the paths are correct and confirmed clicking "Save and Exit".

In the error.log is this reference to ReviveInjector_x64.exe:
```
Exception in thread "main" java.io.IOException: Cannot run program "D:\Program Files\Revive\Revive\ReviveInjector_x64.exe" (in directory "D:\Program Files\Revive\Revive"): CreateProcess error=2, The system cannot find the file specified
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
	at com.cabbagedevelopment.condor2helper.CondorLauncher.launch(CondorLauncher.kt:38)
	at com.cabbagedevelopment.condor2helper.MainApplicationKt.main(MainApplication.kt:30)
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the file specified
	at java.lang.ProcessImpl.create(Native Method)
	at java.lang.ProcessImpl.<init>(ProcessImpl.java:386)
	at java.lang.ProcessImpl.start(ProcessImpl.java:137)
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
	... 2 more
```

Poking around the Revive install folder I don't see a ReviveInjector_x64.exe file anywhere. The only contents of D:\Program Files\Revive\Revive\ are the x64 and x86 sub dirs. (Confirmed this is Revive v1.9.1.)

I tried moving the contents of ...\Revive\Revive\x64 to ...\Revive\Revive and renaming ReviveInjector.exe to ReviveInjector_x64.exe. That results in an empty error.log and suspended Condor2 object in Windows Task Manager. (screenshot 
https://drive.google.com/open?id=1rkT91WUYY2zk33NZl_upD5wvl_4QYYlx)

**Comment by TheGreatCabbage** [comment]
Sorry for the late response; I'm currently rather busy, but I should be able to look into this in the next couple of weeks.

In the meantime, could you tell me which version of Windows you're using, and whether it's 64-bit or 32-bit? Thanks!

**Comment by glenbecker** [comment]
No worries. :)
64-bit, Win 10 Home 1903, OS-build 18362.476

**Comment by TheGreatCabbage** [comment]
This should now be fixed in version `1.0.4`. 

To check whether it works:

- Install the new version from the [Releases](https://github.com/TheGreatCabbage/condor2-revive-helper/releases) page.
- Open the Revive Helper and click "Save and exit".
- Open the desktop shortcut or click a "Join (VR)" link from the Server List.

Let me know if it still has problems. Thanks!

**Comment by glenbecker** [comment]
Sam, 104 is good to go!
Thanks!
